.appveyor.yml

```diff
@@ -8,7 +8,11 @@ configuration:  # a trick to construct a build matrix with multiple Python versi
 environment:
   matrix:
     - COMPILER: MSVC
+      TASK: python
+      CONDA_ENV: test-env
```

Comment:
I don't think we need `CONDA_ENV` in build matrix. Set it like `PYTHON_VERSION`
https://github.com/microsoft/LightGBM/blob/dbc3d05c9c6f89d296a29b73a85a847c818892ad/.appveyor.yml#L19

Reply:
ah good idea